### PR TITLE
removed IE8 hack

### PR DESCRIPTION
**Pull request by sarveshwar-gavhane**
Issue - https://github.com/adaptlearning/adapt_framework/issues/1601

### CODE REVIEW

File: src/core/js/startController.js [hunk 1]
```diff
@@ -17,12 +17,7 @@ define([
 
             if ('replaceState' in window.history) {
```
Comment (moloko) [inline src/core/js/startController.js]: I don't think the `if()` statement is required any more

Comment (oliverfoster) [inline src/core/js/startController.js]: +1

File: src/core/js/startController.js [hunk 2]
```diff
@@ -14,16 +14,7 @@ define([
             if (!this.isEnabled()) return;
 
             var hash = this.getStartHash();
```
Comment (moloko) [inline src/core/js/startController.js]: the local variable `hash` is now redundant, might as well delete this line and change the following line to:
```js
window.history.replaceState('', '', this.getStartHash());
```

**Comment by moloko** [comment]
Please leave this unmerged until we are ready to start with the v3 changes!

**Pull request merged**